friend.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const friendSchema = new Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  age: {
+    type: Number,
+    required: true,
+  },
+  createdOn: {
```

Comment:
We can set the default value for `createdOn` to `Date.now`. That way we don't need all that logic in our route controller to set the date ourselves.

```
createdOn: { type: Date, default: Date.now }
```